q2_fondue/tests/test_scraper.py

```diff
@@ -226,6 +226,51 @@ def test_find_accession_ids_special_cases_three_commas(self):
         obs_proj = _find_accession_ids(txt_diff, 'bioproject')
         self.assertListEqual(sorted(exp_proj), sorted(obs_proj))
 
+    def test_find_accession_ids_special_cases_hyphen_one_digit(self):
+        # example inspired by 10.1038/s41467-019-13036-1
+        txt_diff = 'Breezer mWGS: PRJEB1479791–5'
+        exp_proj = ['PRJEB1479791', 'PRJEB1479792', 'PRJEB1479793',
+                    'PRJEB1479794', 'PRJEB1479795']
+        obs_proj = _find_accession_ids(txt_diff, 'bioproject')
+        self.assertListEqual(sorted(exp_proj), sorted(obs_proj))
+
+    def test_find_accession_ids_special_cases_hyphen_two_digits(self):
+        # example inspired by 10.1038/s41467-019-13036-1
+        txt_diff = 'Scott mWGS: PRJEB1479846–50'
+        exp_proj = ['PRJEB1479846', 'PRJEB1479847', 'PRJEB1479848',
+                    'PRJEB1479849', 'PRJEB1479850']
+        obs_proj = _find_accession_ids(txt_diff, 'bioproject')
+        self.assertListEqual(sorted(exp_proj), sorted(obs_proj))
+
+    def test_find_accession_ids_special_cases_hyphen_three_digits(self):
```

Comment:
Could you please add a test for when the ID rolls over 100? For example, `PRJEB1479098–105`?

Reply:
thanks for spotting this. It was actually not running correctly. I added and corrected it now. Let me know what you think.

Reply:
Awesome, thanks!